docs/usage/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting Renovate
+
+Learn how to troubleshoot problems with Renovate, where to find the logging output, and how to get help if needed.
+
+## Getting the logs
+
+Renovate logs the most recent jobs.
+Where you can find the logs depends on how you're running Renovate.
+
+### GitHub Hosted app
+
+Each pull request from Renovate contains a link to the WhiteSource Renovate Dashboard in the PR body text.
+
+The text you're looking for is:
+
+> This PR has been generated by WhiteSource Renovate. View repository job log here.
+
+Click on the blue text "here" to go to the WhiteSource Renovate Dashboard.
+Sign in with your GitHub or GitLab account.
+Once you're logged in, you can see the logs for the Renovate jobs on your repository.
+
+Renovate only keeps the logs for the last 3 days.
+
+When you've clicked on a recent job, you'll be able to select a debug level that you care about.
+For a full overview, make sure you select the DEBUG log level.
+
+### Selfhosted
+
+TODO: Explain log directory structure. Will probably be different depending on self-hosted platform...
+
+TODO: Explain how long logs are kept by default on differnet platforms.
+
+TODO: Explain how to toggle log level: <https://docs.renovatebot.com/examples/self-hosting/#about-the-log-level-numbers>
```

Comment:
```suggestion
The easiest way to gather logs from Renovate for any platform is to use the default logging to `stdout`/console.
By default, Renovate will log in a human-readable format at `INFO` level.

For troubleshooting it's recommended to increase logging to `DEBUG` level by adding `LOG_LEVEL=debug` to your environment variables before invoking Renovate.

If your Renovate logs are being processed by a log service before you access them, you may find it better to have Renovate output logs in JSON format instead so that they can be reliably parsed and filtered.
This can be achieved by adding `LOG_FORMAT=json` to your environment variables before invoking Renovate.
```